language/documentation/book/src/overview.md

```diff
@@ -27,8 +27,8 @@ Move is a next generation language for secure, sandboxed, and formally verified
     icon="img/placeholder.svg"
     iconDark="img/placeholder-dark.svg"
     overlay="Play with Move directly as you create coins with the language"
-    title="First Tutorial: Creating Coins"
-    to="/docs/move/move-tutorial-creating-coins"
+    title="Move Tutorial"
+    to="https://github.com/diem/move/tree/main/language/documentation/tutorial"
```

Comment:
I don't think the tutorial lives in the `/docs/`, therefore using an absolute path here. But I am not sure how this will get rendered: https://github.com/666lcz/move/blob/02-16-update-tutorial/language/documentation/book/src/overview.md

Reply:
If you want to preview it locally you'll need `mdbook` installed. After that you can preview local changes by running `mdbook serve --open` and that should open a browser with the current changes. 

In order to update the website you'll need to rebuild the mdbook (I usually run `mdbook build` for this and make sure the generated files are in the correct location/submitted to the `gh-pages` branch of this repo). More information on `mdbook` can be found here: https://rust-lang.github.io/mdBook/guide/creating.html